### ISSUE TITLE
Reproducing PopQA Results with qwen2.5-1.5b-instruct: Discrepancy in F1 Scores

### ISSUE THREAD

**Issue by JiaangL** [oneal2000/PRAG#2]
Thanks for your valuable work and for open-sourcing this implementation! I'm attempting to reproduce the PopQA results from Table 1 using the `qwen2.5-1.5b-instruct` model. I encounter some unexpected metric values after executing the command in `configs/popqa_qwen-1.5b-instruct.sh`. `results.txt` contains the following contents:

> em	0.0067
f1	0.0999
prec	0.0649
recall	0.3083
{
    "model_name": "qwen2.5-1.5b-instruct",
    "max_new_tokens": 20,
    "dataset": "popqa",
    "data_type": null,
    "with_cot": false,
    "sample": 300,
    "augment_model": "qwen2.5-1.5b-instruct",
    "num_train_epochs": 2,
    "learning_rate": 0.0003,
    "inference_method": "combine",
    "lora_rank": 2,
    "lora_alpha": 32
}

It seems strange since `f1=0.0999` is the result of Regular RAG according to Table1. Could you help me check what is wrong? I would greatly appreciate guidance on potential misconfigurations in parameters or any known issues. Thank you for your time and assistance!

**Comment by oneal2000** [comment]
Did you run the other datasets and models using the same method? Did you encounter similar issues?

If not, try reinitializing the base LoRA and see if that helps.

**Comment by ge95net** [comment]
Hello,

My result is the same as yours. Have you found the problem and the solution?

**Comment by wangxu-1** [comment]
"Hi, I noticed that the Combine results are identical to the Standard RAG results when running experiments with Llama-1B. Have you encountered this issue before?"

**Comment by JiaangL** [comment]
@ge95net @wangxu-1 
Sorry for the late reply. I haven't followed up on this project since encountering these issues. While I didn't test it on Llama, I observed the same phenomenon on Qwen: the "Combine" results were identical to the Standard RAG results.
